includes/admin/class-sensei-learners-main.php

```diff
@@ -487,12 +488,11 @@ protected function get_row_data( $item ) {
 				$column_data = apply_filters(
 					'sensei_learners_main_column_data',
 					array(
-						'title'            => '<strong><a class="row-title" href="' . esc_url( admin_url( 'user-edit.php?user_id=' . $user_activity->user_id ) ) . '" title="' . esc_attr( $a_title ) . '">' . esc_html( $title ) . '</a></strong>',
+						'title'            => '<strong><a class="row-title" href="' . esc_url( admin_url( 'user-edit.php?user_id=' . $user_activity->user_id ) ) . '" title="' . esc_attr( $a_title ) . '">' . esc_html( $title ) . '</a></strong><div class="row-actions visible">' . implode( ' | ', $row_actions ) . '</div>',
```

Comment:
I believe these should only show when hovering the row, so the`visible` can be removed from `row-actions`.

Reply:
Good catch! https://github.com/Automattic/sensei/pull/3404/commits/4f11e53c4153fdea6623008e72f2fb93990f8c37